src/backend/export/outputVendors/googleSheets/googleSheets.ts

```diff
@@ -10,7 +10,7 @@ import * as googleSheets from './googleSheetsInternalAPI';
 import { appendToSpreadsheet } from './googleSheetsInternalAPI';
 
 const GOOGLE_SHEETS_DATE_FORMAT = 'DD/MM/YYYY';
-const DEFAULT_SHEET_NAME = '_hiuvi';
+const DEFAULT_SHEET_NAME = '_caspion';
```

Comment:
This is a breaking change for people who are already using the integration with google sheets, but I think it is worth doing it now and not leaving this old reference in the code.

Reply:
Yes... but we will need to stop breaking the app someday 😉